Mars_Application_UserStory2_TestAutomation/StepDefinitions/NegativeInvalidStepDefinitions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using TechTalk.SpecFlow;
+using OpenQA.Selenium.Interactions;
+using System;
+using static System.Net.Mime.MediaTypeNames;
+using Mars_Application_UserStory2_TestAutomation.Pages;
+using TechTalk.SpecFlow.Assist;
+using Assert = Xunit.Assert;
+
+namespace Mars_Application_UserStory2_TestAutomation.StepDefinitions
+{
+    [Binding]
+    public class NegativeInvalidStepDefinitions
+    {
+        private readonly IWebDriver driver;
+        private readonly SignUpPage SignUpPageObj = new SignUpPage();
+        private readonly Login LoginPageObj = new Login();
+
+        public NegativeInvalidStepDefinitions(IWebDriver driver)
+        {
+            this.driver = driver;
```

Comment:
please create a common driver class which will initialise the driver, I see you created a hooks class which is good but a better approach would be to extend the Step and Page classes to Common driver class NegativeInvalidStepDefinitions : CommonDriver, In the Hook Class create the instance of Common driver and then class the initialise method for example 

   public class CommonDriver
    {
        public IWebDriver Initialise()
        {

            ChromeOptions options = new ChromeOptions();

            // Set window size to 90% of the screen size
            options.AddArgument("force-device-scale-factor=1");

            return new ChromeDriver(options);
        }

        public static void CloseDriver(IWebDriver driver1)
        {
            if (driver1 != null)
            {
                driver1.Quit();
                driver1 = null;
            }
        }
    }
 and in the Hooks class



public class SpecFlowHooks
{
    private static CommonDriver commonDriver;

    [BeforeTestRun] 
    public static void BeforeTestRun()
    {
        commonDriver = new CommonDriver();
        commonDriver.Initialise();
    }
}